wpilib/wpilib/robotcontroller.py

```diff
@@ -0,0 +1,197 @@
+# validated: 2017-12-24 DS 8744511f1d9b edu/wpi/first/wpilibj/RobotController.java
+#----------------------------------------------------------------------------
+# Copyright (c) FIRST 2017. All Rights Reserved.
+# Open Source Software - may be modified and shared by FRC teams. The code
+# must be accompanied by the FIRST BSD license file in the root directory of
+# the project.
+#----------------------------------------------------------------------------
+
+import hal
+
+__all__ = ['RobotController']
+
+
+class RobotController:
+    
+    @staticmethod
+    def getFPGAVersion() -> int:
```

Comment:
yay for not supporting 3.4

Reply:
Function annotations were introduced in 3.0.